content/en/docs/tasks/administer-cluster/enabling-endpoint-slices.md

```diff
@@ -26,6 +26,7 @@ network endpoints, they will be split into multiple smaller Endpoint Slice
 resources instead of a single large Endpoints resource.
 
 ## Enabling Endpoint Slices
+
 As an alpha feature, Endpoint Slices are not enabled by default in Kubernetes.
```

Comment:
hello @robscott . 
As I recall, this task page is evolving. What do you think about adding:
- a feature state shortcode at around line 30 or in the overview section?

{{< feature-state for_k8s_version="v1.16" state="alpha" >}}

- a note shortcode to replace the content in the new section:

{{< note >}}
Although Endpoint Slices may eventually replace Endpoints, many Kubernetes
components still rely on Endpoints. For now, enabling Endpoint Slices should be
seen as an addition to Endpoints in a cluster, not a replacement for them.
{{< /note >}}

Reply:
Hey @kbhawkey, I really like this idea, thanks! I've pushed an update with those changes. As I'm thinking about this a bit more, this entire task is only relevant when Endpoint Slices are in alpha, and once they graduate to beta and are enabled by default (hopefully in 1.17), the task should be removed. Do I need to do anything to make that more clear in this task or should I just add a PR to remove the task once EndpointSlices reach beta?

Reply:
Hi @robscott . I suppose you can decide whether to update or remove this page when Endpoint Slices reaches beta. Of course you can add more content or change the content anytime.